packages/flutter_tools/lib/src/resident_web_runner.dart

```diff
@@ -2,40 +2,179 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: implementation_imports
 import 'dart:async';
 
+import 'package:build_daemon/client.dart';
+import 'package:build_daemon/data/build_status.dart';
+import 'package:build_daemon/data/build_target.dart';
+import 'package:build_daemon/data/server_log.dart';
+import 'package:dwds/service.dart';
 import 'package:meta/meta.dart';
-import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf/src/request.dart';
+import 'package:vm_service_lib/vm_service_lib.dart' as vmservice;
+
+/// SRC IMPORTS
```

Comment:
I reduced the list to what I currently need, generally:

* Chrome and friends - I think there was already talk of making the reusable.
* The DevTools launcher - this may need an additional argument so we can adjust the launch 
* daemonPort - I think this is in build_daemon already? Or maybe it should be.
* The Configuration & ReloadConfiguration classes are needed to configure the WebDevServer
* The debugService and devhandler to get the vmservice information